### PR TITLE
ergoCubSN001 - neck files updated

### PR DESCRIPTION
**Pull request by davidetome**
This PR updates XML files for the neck following tests on the setup

⚠️ This mod applies only to the pitch and roll of the neck, other joints of the head have to be calibrated as soon as available

cc @pattacini @sgiraz

### CODE REVIEW

File: ergoCubSN001/hardware/motorControl/head-eb20-j0_1-mc.xml [hunk 1]
```diff
@@ -2,69 +2,74 @@
...
-    <xi:include href="../../hardware/mechanicals/head-eb20-j0_1-mec.xml" />
-    <xi:include href="./head-eb20-j0_1-mc_service.xml" />
+    <xi:include href="../../hardware/electronics/head-eb21-j0_1-eln.xml" />
+    <xi:include href="../../hardware/mechanicals/head-eb21-j0_1-mec.xml" />
+    <xi:include href="./head-eb21-j0_1-mc_service.xml" />
 
-    <!-- joint number                           0                   1                   -->
-    <!-- joint name                             neck-pitch          neck-roll           -->
+    <!-- joint number                           0                -->
+    <!-- joint name                             neck-yaw         -->         <!-- j2 is right-eye, j3 is left-eye -->
```
Comment (pattacini) [inline ergoCubSN001/hardware/motorControl/head-eb20-j0_1-mc.xml:13]: Probably a leftover.
@davidetome, could you fix it up?

**Review by pattacini — APPROVED** [review]
Thanks @davidetome 👍🏻 
Awaiting the CI before merging.

**Comment by pattacini** [comment]
CI still fails:

```console
WARNING] ServiceParser::convert(): P3P is not a legal string for eObrd_connector_t
[WARNING] ServiceParser::parse_port_conn(): CONN:P3P is not a legal string for a eObrd_connector_t
[WARNING] ServiceParser::parse_actuator_port(): CONN:P3P is not a legal string for a pwm connector port
[ERROR] ServiceParser::check_motion() PROPERTIES.JOINTMAPPING.actuator.port not valid for item 0
[ERROR] ServiceParser::parseService() gets same errors parsing SERVICE MC group
[ERROR] embObjMC  BOARD head-eb20-j0_1 (IP 10.0.1.20)  cannot parse service
[ERROR] BOARD head-eb20-j0_1 (IP 10.0.1.20)  Missing motion control parameters in config file
[ERROR] |yarp.dev.PolyDriver|head-eb20-j0_1-mc| Driver <embObjMotionControl> was found but could not open
```

This is due to the fact that we haven't integrated yet the new FW.
Thus, I'll keep this PR in draft and come back to it once the FW is updated.

**Comment by pattacini** [comment]
> Thus, I'll keep this PR in draft and come back to it once the FW/parser is updated.

Awaiting the following PR to be merged before putting this in ready for review:
- https://github.com/robotology/icub-firmware-shared/pull/87

**Review by sgiraz — APPROVED** [review]
Hi @davidetome, the PR is ok!

**Review by marcoaccame — APPROVED** [review]
OK

**Review by AntonioConsilvio — APPROVED** [review]

**Comment by pattacini** [comment]
Required upstream PR merged ✅

I'm rebuilding the docker image used by the CI via https://github.com/robotology/robots-configuration/actions/runs/6198027673.

Once done, I'll put this PR in ready-for-review.

**Pull request merged**